lib/taskana-core-test/src/test/java/acceptance/workbasket/get/GetWorkbasketAccTest.java

```diff
@@ -103,11 +103,12 @@ void should_ReturnWorkbasketPermissions_When_IdIsValidAndUserHasPermissions() {
     List<WorkbasketPermission> permissions =
         workbasketService.getPermissionsForWorkbasket(defaultWorkbasketSummary.getId());
 
-    assertThat(permissions).hasSize(4);
-    assertThat(permissions.contains(WorkbasketPermission.READ)).isTrue();
-    assertThat(permissions.contains(WorkbasketPermission.OPEN)).isTrue();
-    assertThat(permissions.contains(WorkbasketPermission.TRANSFER)).isTrue();
-    assertThat(permissions.contains(WorkbasketPermission.APPEND)).isTrue();
+    assertThat(permissions)
```

Comment:
Please change assertion to "containsExactlyInAnyOrder". You don't need to check the size.